airflow/providers/amazon/aws/triggers/eks.py

```diff
@@ -46,57 +43,35 @@ def __init__(
         waiter_max_attempts: int,
         aws_conn_id: str,
         region: str | None = None,
+        region_name: str | None = None,
     ):
-        self.cluster_name = cluster_name
-        self.fargate_profile_name = fargate_profile_name
-        self.waiter_delay = waiter_delay
-        self.waiter_max_attempts = waiter_max_attempts
-        self.aws_conn_id = aws_conn_id
-        self.region = region
-
-    def serialize(self) -> tuple[str, dict[str, Any]]:
-        return (
-            self.__class__.__module__ + "." + self.__class__.__qualname__,
-            {
-                "cluster_name": self.cluster_name,
-                "fargate_profile_name": self.fargate_profile_name,
-                "waiter_delay": str(self.waiter_delay),
-                "waiter_max_attempts": str(self.waiter_max_attempts),
-                "aws_conn_id": self.aws_conn_id,
-                "region": self.region,
-            },
+        if region is not None:
```

Comment:
I tend to be cautious with checks like this, because it'll trigger on empty string as well (and empty array, empty dict, etc.), and I've been bitten by this several times.
Here it's not a problem, but I prefer to be more explicit in general.